examples/index.html

```diff
@@ -564,7 +589,7 @@
 
 		}
 
-		</script>
+		</script><div class="marquee"><p>refresh_fix_on_some_blacklisted_android_chipsets</p></div></article>
```

Comment:
I've not disabled blacklisting on stock browser, it renders whatever you throw at it.
Stock browser doesn't have blacklist.
That's the point. Hundreds of millions of users would benefit.